### PR TITLE
Docs/strategies overview update

### PR DESCRIPTION
**Pull request by lvisintini**
Updated the table for the strategies in the first Axelrod Tournament
It should now be easy to determine which strategies have yet to be implemented

### CODE REVIEW

**Comment by marcharper** [comment]
:+1:

File: docs/overview_of_strategies.rst [hunk 1]
```diff
@@ -6,32 +6,49 @@ Axelrod's first tournament
...
+| `Feld`_                  | Feld (author's name)                      | :code:`Feld`             |
++--------------------------+-------------------------------------------+--------------------------+
+| `Joss`_                  | Joss (author's name)                      | :code:`Joss`             |
++--------------------------+-------------------------------------------+--------------------------+
+| `Tullock`_               | Tullock (author's name)                   | :code:`Tullock`          |
++--------------------------+-------------------------------------------+--------------------------+
+| `Unnamed Strategy`_      | Unnamed Strategy (by a Grad Student in    | Not Implemented          |
+|                          | Political Science)                        |                          |
++--------------------------+-------------------------------------------+--------------------------+
+| `Random`_                | Random                                    | :code:`Random`           |
```
Comment (lvisintini) [inline docs/overview_of_strategies.rst]: Don't really know why this like does not take you to the appropriate section in the page. Any ideas what I'm missing?

Comment (marcharper) [inline docs/overview_of_strategies.rst]: Seems like github isn't parsing the file correctly?

Comment (drvinceknight) [inline docs/overview_of_strategies.rst]: I think the error @lvisintini meant was that upon building the Random strategy does not link correctly. I thought it was not in the file but checked and I was wrong. I'm not sure what is happening here.

**Comment by meatballs** [comment]
I've fixed this one locally by using the full ref syntax. I think it was some weird reserved word problem with the word 'Random' itself.

I'll delete this PR, create a new branch from it, add my fix and submit a new PR.

**Comment by lvisintini** [comment]
Cool,
I also thought that could have been the case, but I have not a chance to
look into the code again since Monday

**Pull request closed without merging**